FrontEnd/src/components/ProfilePage/FormComponents/GeneralInfo.js

```diff
@@ -248,15 +233,19 @@ const GeneralInfo = (props) => {
                             value={user.edrpou}
                             error={edrpouError}
                         />
-                        <MultipleSelectChip
+                        {!isLoading && 
```

Comment:
Should we show the loading state? Do we have any kind of error state?

Reply:
I decided not to implement this for now because I remember my teammate wanted to complete this task. I only added this condition, because, otherwise I might get an error.